configuration/configuration.go

```diff
@@ -535,7 +516,7 @@ func experimental(key string) {
 
 func doneSquash(value string) string {
```

Comment:
Is this function need anymore? If someone sets the value to a value that does not exist this should be handled within mob done squash or isn't it that way?

Reply:
Tried removing it. But it is what handles turning "garbage" into a default. Could code that differently, but I did not come up with a simpler solution, so I left it.

Reply:
Ok fine for me